app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -58,8 +58,8 @@
   <dimen name="text_input_interaction_item_conversation_view_margin_start">144dp</dimen>
   <dimen name="profile_chooser_fragment_margin_top_profile_not_added">124dp</dimen>
 
-  <dimen name="continue_interaction_item_exploration_view_margin_end">112dp</dimen>
-  <dimen name="continue_interaction_item_exploration_view_margin_start">112dp</dimen>
+  <dimen name="continue_interaction_item_exploration_view_padding_end">112dp</dimen>
+  <dimen name="continue_interaction_item_exploration_view_padding_start">112dp</dimen>
```

Comment:
Why is this getting changed? This PR is not using this change anywhere.

Reply:
Done.